src/features/counterfactual/CounterfactualForm.tsx

```diff
@@ -141,15 +164,12 @@ export const CounterfactualForm = ({
             </ErrorMessage>
           )
         )}
-
         {submitError && (
           <Box mt={1}>
             <ErrorMessage error={submitError}>Error submitting the transaction. Please try again.</ErrorMessage>
           </Box>
         )}
-
         <Divider className={commonCss.nestedDivider} sx={{ pt: 3 }} />
-
```

Comment:
Waren gute, diese.

Reply:
This also happened another time but I can't figure out how they are deleted. Afaik we don't have any lint or prettier rules to remove these...